hot-reload-runtime-jvm/src/main/kotlinUI/org/jetbrains/compose/hotReloadUI/widgets/TimeAgoText.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2024-2025 JetBrains s.r.o. and Compose Hot Reload contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+ */
+
+package org.jetbrains.compose.hotReloadUI.widgets
+
+import androidx.compose.foundation.text.BasicText
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorProducer
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.sp
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.isActive
+import java.time.Duration
+import java.time.Duration.between
+import java.time.Instant
+import kotlin.time.Duration.Companion.milliseconds
+
+@Composable
+internal fun TimeAgoText(
```

Comment:
I think `TimeAgoText` is actually one of the reasons why the tests are failing. Maybe we can adapt the use of `VirtualTimeState` here somehow to guarantee what will be displayed in tests?